abstutil/src/cli.rs

```diff
@@ -18,7 +18,7 @@ impl CmdArgs {
             last_fp_note: RwLock::new(None),
         }))
         .unwrap();
-        log::set_max_level(log::LevelFilter::Trace);
+        log::set_max_level(log::LevelFilter::Debug);
```

Comment:
👀  I can revert this and just keep my change locally if this is controversial, but I *generally* don't want all the log spam created by trace-level logging, and in the rare moments I do, I'm happy to flip it back uncommited.

Reply:
Nah, this is fine to keep. If you have requests for logging, https://github.com/dabreegster/abstreet/issues/262 collects ideas